lib/jekyll-admin/directory.rb

```diff
@@ -0,0 +1,62 @@
+module JekyllAdmin
+  class Directory
+    attr_reader :path
+
+    include Enumerable
+    include JekyllAdmin::URLable
+    include JekyllAdmin::APIable
+
+    TYPE = 'directory'
+
+    def initialize(path, args={})
```

Comment:
That's good to know, but I need to pass these arguments [here](https://github.com/jekyll/jekyll-admin/blob/455329fa8d811d0b7a2b11dc032b2a1c7643d75e/lib/jekyll-admin/directory.rb#L58) recursively. Is there a shortcut for that in Ruby?